packages/rpc-json-parsed/src/token-accounts.ts

```diff
@@ -0,0 +1,17 @@
+import type { Address } from '@solana/addresses';
+import type { TokenAmount } from '@solana/rpc-types';
+
+type TokenAccountState = 'initialized' | 'uninitialized' | 'frozen';
+
+export type TokenAccount = Readonly<{
+    mint: Address;
+    owner: Address;
+    tokenAmount: TokenAmount;
+    delegate?: Address;
+    state: TokenAccountState;
+    isNative: boolean;
+    rentExemptReserve?: TokenAmount;
+    delegatedAmount?: TokenAmount;
+    closeAuthority?: Address;
+    extensions?: unknown[];
```

Comment:
Why is my beloved key sort linter not failing on this? :)

Reply:
I hate that thing.

Reply:
The place I copied it from has it in that order too so I think that's been ignoring things for a while, not sure why!

Reply:
IIRC, that linter rule only checks "real" JavaScript objects, not object types.